README.md

```diff
@@ -95,6 +95,25 @@ The following are the parameters for creating the cloudformation template:
 
 [![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/colored.png)](#instructions)
 
+## Tagging
```

Comment:
Can this be a seperate markdown file under docs . The worry is that the readme is just increasing in size

Reply:
sure